android-exoplayer/src/main/java/com/brentvatne/exoplayer/ReactExoplayerView.java

```diff
@@ -398,6 +432,12 @@ public void run() {
                 initializePlayerControl();
                 setControls(controls);
                 applyModifiers();
+
+                //Use Media Session Connector from the ExoPlayer library to enable MediaSession Controls in PIP.
+                MediaSessionCompat mediaSession = new MediaSessionCompat(getContext(), "tag");
+                MediaSessionConnector mediaSessionConnector = new MediaSessionConnector(mediaSession);
+                mediaSessionConnector.setPlayer(player, null);
+                mediaSession.setActive(true);
```

Comment:
we may need to clean up the session later in the lifecycle.

Reply:
https://github.com/react-native-community/react-native-video/pull/1780